src/cmd/create.rs

```diff
@@ -63,15 +73,16 @@ pub struct Sharded {
     recovery_threshold: u8,
 
     #[structopt(long)]
-    /// Use space separated seed words to create the wallet
-    seed: bool,
+    /// Use a BIP39 or mobile app seed phrase to generate the wallet keys
+    /// (bip39, mobile)
+    seed: Option<SeedType>,
```

Comment:
If you want to always define a type, use the structopt `default_value = "mobile"`. That way you don't need an Option here at all and avoid having expects or unwraps everywhere

Reply:
But `--seed` is a optional on the cmdline. You can run `create` without it and the wallet just generates its own entropy.

Reply:
ah shoot, yeah, nvm this particular nit